allvers/1998-01-26-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,11 @@
+Date: 1998-01-26 or earlier
```

Comment:
Possibly 1997 if the changed date on the website is correct (and it could be that it wasn't updated):

```
Updated: 4 Aug 1997 tower 
```

Reply:
Updated in 69f90f4b7875e5b2249f2aedae0e966e5589d111